HorizonApache/src/main/java/com/hubspot/horizon/apache/ApacheHttpClient.java

```diff
@@ -141,7 +151,16 @@ private HttpResponse executeWithRetries(HttpRequest request, Options options, in
 
       final HttpResponse response;
       try {
-        apacheResponse = apacheClient.execute(apacheRequest);
+        if (config.isSocksProxied()) {
+          InetSocketAddress socksaddr = new InetSocketAddress(config.getSocksProxyHost(), config.getSocksProxyPort());
+          HttpClientContext context = HttpClientContext.create();
+          context.setAttribute("socks.address", socksaddr);
+          LOG.info("Sending http request to {} via proxy @{}", request.getUrl(), config.getSocksProxyHost());
```

Comment:
This could be quite noisy. Is it necessary to log this on every request? Could it be `debug`?